tests/instancetype/instancetype.go

```diff
@@ -711,6 +723,8 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 		createAndValidateVirtualMachine := func() {
 			By("Creating the VirtualMachine")
 			var err error
+			libvmi.WithRunStrategy(virtv1.RunStrategyAlways)(vm)
+			vm.Spec.Running = nil
```

Comment:
I don't like comments. They typically mean that the code is not clear enough.

However, this is such a case where a comment would help me understand why you erase `vm.Spec.Running` and overwrite `vm.Spec.RunStrategy`. I'd appreciate a TODO explaining when we can drop this oddity.

If you disagree, maybe an explanation in the commit message may be enough.

Reply:
Yes, you are right;  this change does introduce some technical debt, which I also don't prefer.
How about setting `vm.Spec.RunStrategy = RunStrategyHalted` instead of `vm.Spec.Running = false` from the beginning?
Please review and let me know your thoughts.

Reply:
This looks good to me, but I'd like to hear @lyarwood.